tests/pytorch/nightly/dlrm.libsonnet

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local common = import "common.libsonnet";
+local timeouts = import "templates/timeouts.libsonnet";
+local tpus = import "templates/tpus.libsonnet";
+local utils = import "templates/utils.libsonnet";
+
+{
+  local command_common = |||
+    git clone https://github.com/taylanbil/dlrm.git -b tpu
+    pip install onnx
+    python dlrm/dlrm_tpu_runner.py \
+      --arch-sparse-feature-size=64 \
+      --arch-mlp-bot=512-512-64 \
+      --arch-mlp-top=1024-1024-1024-1 \
+      --arch-interaction-op=dot \
+      --lr-num-warmup-steps 10 \
+      --lr-decay-start-step 10 \
+      --num-batches=1000 \
+      --data-generation="random" \
+      --numpy-rand-seed=727 \
+      --print-freq 100 \
+      --num-indices-per-lookup=100 \
+      --use-tpu \
+      --metrics-debug \
+      --num-indices-per-lookup-fixed \
+  |||,
+  local dlrm = common.PyTorchTest {
```

Comment:
I think you should add:

`schedule: "0 21 * * *"`

This will offset a bit from some other tests that all start at `19`

Reply:
oh nice pro tip, thanks!